packages/react/scroll-area/src/ScrollArea.tsx

```diff
@@ -693,7 +695,11 @@ type ScrollAreaViewportPrimitive = Polymorphic.ForwardRefComponent<
 
 const ScrollAreaViewport = React.forwardRef(function ScrollAreaViewport(props, forwardedRef) {
   return useNativeScrollArea() ? (
-    <Primitive selector={getSelector(VIEWPORT_NAME)} {...props} ref={forwardedRef} />
+    <Primitive
+      {...props}
+      selector={props.selector || getSelector(VIEWPORT_NAME)}
```

Comment:
I'm sure there's a reason but can't this one also be a default arg like the others? Shouldn't it also get passed to `ScrollAreaViewportImpl`?

Reply:
`ScrollAreaViewportImpl` receives all the `{...props}` and does the `selector = getSelector` thing internally itself.